contracts/provider/assembly/index.ts

```diff
@@ -0,0 +1,216 @@
+import {
+  u128,
+  storage,
+  context,
+  ContractPromiseBatch,
+  PersistentUnorderedMap,
+  logging,
+} from "near-sdk-as";
+import { onlyOwner } from "../../ownable/assembly";
+export {
+  transferOwnership,
+  renounceOwnership,
+  owner,
+} from "../../ownable/assembly";
+
+export const DefaultDivisor = u128.from("416000000000"); // yocto-Near per nano-second
```

Comment:
Yeh, so this is actually the default number of yocto-NEAR per nano-second of session time. That isn't particularly useful for humans, but makes the math simpler. I think basically the clients will do the conversion to nano-seconds behind the scenes.